rocAL/rocAL/include/augmentations/geometry_augmentations/node_flip.h

```diff
@@ -25,23 +25,22 @@ THE SOFTWARE.
 #include "parameter_vx.h"
 #include "parameter_factory.h"
 
-class FlipNode : public Node
-{
+class FlipNode : public Node {
 public:
     FlipNode(const std::vector<Tensor *> &inputs, const std::vector<Tensor *> &outputs);
     FlipNode() = delete;
-    void init(int flip_axis);
-    void init(IntParam *flip_axis);
+    void init(int h_flag, int v_flag);
+    void init(IntParam *h_flag_param, IntParam *v_flag_param);
     unsigned int get_dst_width() { return _outputs[0]->info().max_shape()[0]; }
     unsigned int get_dst_height() { return _outputs[0]->info().max_shape()[1]; }
     vx_array get_src_width() { return _src_roi_width; }
     vx_array get_src_height() { return _src_roi_height; }
-    vx_array get_flip_axis() { return _flip_axis.default_array(); }
+    vx_array get_flip_axis() { return _horizontal.default_array(); }
 protected:
     void create_node() override;
     void update_node() override;
 private:
-    int _axis;
-    ParameterVX<int> _flip_axis;
-    constexpr static int   FLIP_SIZE [2] =  {0, 2};
-};
\ No newline at end of file
+    ParameterVX<int> _horizontal, _vertical;
+    constexpr static int HORIZONTAL_RANGE[2] = {0, 1};
+    constexpr static int VERTICAL_RANGE[2] = {0, 1};
```

Comment:
@fiona-gladwin Why has this value changed?

Reply:
RPP has been changed to take 2 separate params for Vertical and Horizontal flip to match DALI. So we need to pass 2 params.